confidence-cloudflare-tail/src/index.js

```diff
@@ -0,0 +1,47 @@
+export default {
+    async tail(events, env, ctx) {
```

Comment:
I would prefer moving the tail folder inside the `confidence-cloudflare-resolver`: I think it is not worthy of a space in the root